lib/tools/adb-commands.js

```diff
@@ -707,8 +707,17 @@ methods.getDeviceProperty = async function (property) {
 };
 
 methods.setDeviceProperty = async function (prop, val) {
+  let apiLevel = await this.getApiLevel();
+  if (apiLevel >= 26 ) {
```

Comment:
@mykola-mokhnach what do you think of this approach ? I would raise an exception after running `.unroot`.